internal/services/network/virtual_network_data_source.go

```diff
@@ -69,6 +69,13 @@ func dataSourceVirtualNetwork() *pluginsdk.Resource {
 					Type: pluginsdk.TypeString,
 				},
 			},
+			"vnet_peerings_address_lists": {
```

Comment:
we don't include "list" in the property name, so this should be
```suggestion
			"vnet_peerings_addresses": {
```

and